CMakeLists.txt

```diff
@@ -70,6 +71,31 @@ find_package(Gettext REQUIRED)
 include_directories(${INTL_INCLUDE_DIRS})
 link_directories(${INTL_LIBRARY_DIRS})
 
+#Helper funciton to find X11-xcb and xcb
+function(local_find_library VAR NAME)
+    find_library("${VAR}" "${NAME}")
+
+    if("${VAR}")
+        message(STATUS "Library found: ${NAME}")
+    else("${VAR}")
+        message(STATUS "Library not found: ${NAME}")
+    endif("${VAR}")
+endfunction(local_find_library)
+
+
+#EGL, X11-xcb, xcb
+#Use these libs if running on Wayland
+if( $ENV{XDG_SESSION_TYPE} STREQUAL "wayland")
```

Comment:
This is very system-dependent and avoids reproducible builds.
Instead of this, use a variable and let the user decide if we should compile with wayland support or not.

Reply:
Done